yarn.lock

```diff
@@ -4902,7 +4907,7 @@
     "@types/webpack-dev-server" "^3"
     webpack-chain "^6.0.0"
 
-"@vueuse/core@^10.7.1":
+"@vueuse/core@10.9.0", "@vueuse/core@^10.7.1":
```

Comment:
To prevent conflicts, don't forget to sync deps versions between XO Lite, XO 6 and XO Core